cloud/helm-chart/README.md

```diff
@@ -0,0 +1,34 @@
+# Install
+
+```
+$ helm install zilla . --namespace zilla --create-namespace --wait #--values your.custom.values.yaml
```

Comment:
Should this example show how to specify file for zilla.yaml too?
(same for upgrade)

Reply:
fixed